kontakt-reservasjonsregister-client/src/main/java/no/difi/meldingsutveksling/ptp/KontaktInfo.java

```diff
@@ -1,31 +1,31 @@
 package no.difi.meldingsutveksling.ptp;
 
 import com.google.common.base.MoreObjects;
-import no.difi.ptp.sikkerdigitalpost.HentPersonerRespons;
-import no.difi.ptp.sikkerdigitalpost.Person;
+import no.difi.ptp.sikkerdigitalpost.*;
 import org.bouncycastle.openssl.jcajce.JcaPEMWriter;
-import org.bouncycastle.util.io.pem.PemWriter;
 
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.io.StringWriter;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.io.StringWriter'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AUselessImportCheck)